server/superdesk/errors.py

```diff
@@ -312,3 +312,33 @@ def ftpUnknownParserError(cls, exception, provider, filename):
             logger.exception("Provider: {} - File: {} unknown file format. "
                              "Parser couldn't be found.".format(provider.get('name', 'Unknown provider'), filename))
         return IngestFtpError(5001, exception, provider)
+
+
+class IngestRssFeedError(SuperdeskIngestError):
```

Comment:
I can move the errors to one of the existing classes, the `IngestApiError` class seems like the best candidate to me?